File: library/src/main/java/com/imangazaliev/circlemenu/MenuControllerListener.java
package com.imangazaliev.circlemenu;

interface MenuControllerListener {

    void onOpenAnimationStart();

    void onOpenAnimationEnd();

    void onCloseAnimationStart();

    void onCloseAnimationEnd();

    void onSelectAnimationStart(CircleMenuButton menuButton);

    void onSelectAnimationEnd(CircleMenuButton menuButton);

    void redrawView();
}

File: library/src/main/java/com/imangazaliev/circlemenu/MenuControllerListener.kt
package com.imangazaliev.circlemenu

internal interface MenuControllerListener {

    fun onOpenAnimationStart()

    fun onOpenAnimationEnd()

    fun onCloseAnimationStart()

    fun onCloseAnimationEnd()

    fun onSelectAnimationStart(menuButton: CircleMenuButton)

    fun onSelectAnimationEnd(menuButton: CircleMenuButton)

    fun redrawView()

}